src/pages/Home/components/Syndromes/index.js

```diff
@@ -519,6 +545,17 @@ const Syndromes = ({
                                 />
                             </InputBlock>
 
+                            <InputBlock>
+                                <label htmlFor="threshold_score">Pontuação limite para match com a sindrome em %</label>
+                                <input
+                                    type="number"
+                                    id="days_period"
+                                    value={syndromeDays}
+                                    onChange={(e) => setSyndromeDays(e.target.value)}
+                                    min="1"
+                                />
```

Comment:
Esse input ta igual ao de Período de dias fazendo que mude os dois ao mesmo tempo, o id dele deveria ser "threshold_score" (o for do label). Eu também colocaria esse InputBlock por último já que pra visualizar e editar ta nessa ordem.

Reply:
Esse problema ja está resolvido